themes/src/main/resources/theme/base/admin/resources/partials/client-detail.html

```diff
@@ -473,6 +473,23 @@
 
         <!-- KEYCLOAK-6771 Certificate Bound Token https://tools.ietf.org/html/draft-ietf-oauth-mtls-08#section-3 -->
         <fieldset data-ng-show="protocol == 'openid-connect'">
+            <div class="form-group">
```

Comment:
This block should be below `<legend>` (l. 493)

Reply:
Fixing

Reply:
All comments to date incorporated